source/vir/src/ast.rs

```diff
@@ -406,6 +407,7 @@ pub enum ComputeMode {
 pub type Expr = Arc<SpannedTyped<ExprX>>;
 pub type Exprs = Arc<Vec<Expr>>;
 #[derive(Debug)]
+#[to_node_impl(name = ">")]
```

Comment:
Just to confrim, is ">" replacing "ExprX" in the VIR log?

Reply:
Yes!